android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -242,6 +304,22 @@ public void rejectEvent(String eventId) {
         Radar.rejectEvent(eventId);
     }
 
+    @ReactMethod
+    public void startTrip(ReadableMap optionsMap) {
+        try {
+            JSONObject optionsObj = RNRadarUtils.jsonForMap(optionsMap);
+            RadarTripOptions options = RadarTripOptions.fromJson(optionsObj);
+            Radar.startTrip(options);
+        } catch (JSONException e) {
+
+        }
+    }
+
+    @ReactMethod
+    public void stopTrip() {
+        Radar.stopTrip();
```

Comment:
wrap in a `try / catch` as well?

Reply:
Don't need to wrap in a `try/catch` because we're not manipulating a `JSONObject` here